backend/app/services/user.py

```
@@ -40,37 +41,43 @@ def _validate_git_info(self, git_info: List[Dict[str, Any]]) -> List[Dict[str, A
+
             try:
                 # Use specific provider's validate_token method with custom domain
                 git_domain = git_item.get("git_domain")
-                validation_result = provider.validate_token(git_item["git_token"], git_domain=git_domain)
-                
+                validation_result = provider.validate_token(plain_token, git_domain=git_domain)
+
                 if not validation_result.get("valid", False):
                     raise ValidationException(
                         f"Invalid {provider_type} token"
                     )
```

Ruff (0.14.4)

Abstract `raise` to an inner function

(TRY301)